jobsdb/jobsdb.go

```diff
@@ -1533,11 +1513,27 @@ type transactionHandler interface {
 }
 
 func (jd *HandleT) createDS(newDS dataSetT, l lock.DSListLockToken) {
```

Comment:
test for the same can be found here: https://github.com/rudderlabs/rudder-server/blob/e0117268aa350d05b7a8fbadce444340ae79cebf/jobsdb/integration_test.go#L875